scripts/setRPATHWrapper.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
that is never called is it ?

Reply:
No but I don't see the harm in having it

Reply:
Oh sorry I misunderstood this comment. Yes you're correct, I'll remove it